src/client/NuGet.Client.Staging/project.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "1.0.0-alpha-1",
+    "buildOptions": {
+        "warningsAsErrors": true
+    },
+    "dependencies": {
+        "NuGet.Protocol.Core.v3": "3.5.0-beta-final"
+    },
+    "frameworks": {
+        "net45": {

```

Comment:
done

---
In reply to: [66830639](https://github.com/NuGet/NuGet.Staging/pull/62#discussion_r66830639) [](ancestors = 66830639)